tests/websocket.rs

```diff
@@ -537,3 +571,71 @@ fn create_test_configuration(admin_port: u16) -> HcHttpGatewayResult<Configurati
         "",
     )
 }
+
+impl AdminCall for AdminWrapper {
```

Comment:
Also temporary, can use the real one once available. I thought it was better to leave a complete placeholder in the code and make a minor mess in the tests that is easier to clean up.